packages/augur-core/source/libraries/ContractDeployer.ts

```diff
@@ -143,7 +145,8 @@ Deploying to: ${networkConfiguration.networkName}
         }
 
         // Proxy Factory & Gnosis Safe
-        if (externalAddresses.ProxyFactory) {
+        if (this.configuration.isProduction || externalAddresses.GnosisSafe) {
+            if (!externalAddresses.GnosisSafe) throw new Error('Must provide GnosisSafe');
```

Comment:
I'm curious why you didn't put this check inside the check below.  I personally like the individual checks like this rather than multi-checks, but it doesn't follow the same style as other sections.

Reply:
I'm just putting it after the first condition everywhere